### DISCUSSION TITLE
not able to deploy Tamagui Next Web app  ( Monorepo )

### DISCUSSION THREAD

**Discussion by prempro42** [vercel/community#2451] [Help] [answered]
### Summary

need to deploy my next.js app -> located in 'apps/next'

**my monorepo** : [tama-test](https://github.com/Prem-Kumar-G/tama-test/tree/main)

**my Build & Development Settings:**  :

![build_cmd](https://github.com/vercel/community/assets/31033009/b11bc9e5-2f28-40d4-abc9-b8c2d8b42f8d)

**Build Failed**  :

![deployment](https://github.com/vercel/community/assets/31033009/09f6aad0-1f86-4ce8-9240-a0de6d9c0ba7)

if i remove root directory, build will be success but 'not found' page is displayed.

 @amyegan  - please help 🙏 me with steps/config to fix this issue

**Comment by amyegan** [comment, marked as answer]
Hi @Prem-Kumar-G! It looks like there is a `vercel.json` file overriding your Install Command setting. If you remove this, the build should stop throwing that error. 

https://github.com/Prem-Kumar-G/tama-test/blob/f95b443ec8908e0c512905a9bc430d07c0b3ce26/apps/next/vercel.json#L2

**Reply by prempro42** [reply]
thanks a lot @amyegan  :) i'm a newbie to vercel, went through documentation and kept installCommand in vercel.json as empty string. 

+ for anyone - it worked with below Build & Development Settings

![chrome-capture-2023-4-16](https://github.com/vercel/community/assets/31033009/cd77343d-843e-4642-95e5-615e56951353)
